### DISCUSSION TITLE
Daily 11/08

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#54] [Daily]
**Con respecto al proyecto:**

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Dificultades o impedimentos.

**Comment by MarielaGomez** [comment]
Estuve adelantando las actividades de los distintos módulos, en especial es de aplicaciones móviles, leyendo apuntes y ayudándome a reforzar por canales tutoriales de youtube

**Comment by BraianTroncoso** [comment]
Hoy tuve clases de python, seguí viendo un par de videos y sigo con otros apuntos similares de acuerdo a android. Voy a asignarme alguna issue para adelantar.

**Comment by Valentino399** [comment]
Hoy me pondré a hacer el diseño de la aplicación en el programa de figma
